src/main/java/edu/kit/ipd/dbis/gui/correlation/CorrelationSuggestions.java

```diff
@@ -0,0 +1,96 @@
+package edu.kit.ipd.dbis.gui.correlation;
+
+import edu.kit.ipd.dbis.org.jgrapht.additions.graph.Property;
+import edu.kit.ipd.dbis.org.jgrapht.additions.graph.PropertyFactory;
+import edu.kit.ipd.dbis.org.jgrapht.additions.graph.PropertyGraph;
+
+import javax.swing.*;
+import javax.swing.text.BadLocationException;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+public class CorrelationSuggestions extends JPopupMenu {
+
+	private final JTextField inputField;
+
+	public CorrelationSuggestions(JTextField inputField) {
+		this.inputField = inputField;
+		updateSuggestions();
+	}
+
+	private void updateSuggestions() {
+		List<String> relations = new ArrayList<>();
+		relations.add("Max");
+		relations.add("Min");
+		relations.add("Least");
```

Comment:
What is the difference between "Min" and "Least"? The correlation package currently does not allow expressions like "Least Pearson 3".

Reply:
"Min" was the smallest correlation, "Least" the correlation closest to 0.